wmfdata/hive.py

```diff
@@ -1,25 +1,45 @@
-import datetime as dt
 import os
-import re
-from shutil import copyfileobj
-import subprocess
+import pwd
 import tempfile
-import warnings
-
 import pandas as pd
 
+from pyhive import hive
+from shutil import copyfileobj
 from wmfdata.utils import (
-    check_kerberos_auth,
     ensure_list
 )
 
+HIVE_URL = 'analytics-hive.eqiad.wmnet'
+KERBEROS_SERVICE_NAME = 'hive'
+
+
 def run_cli(
-  commands, format = "pandas", heap_size = 1024, use_nice = True,
-  use_ionice = True
+    commands, format="pandas", heap_size=1024, use_nice=True,
+    use_ionice=True
 ):
     """
-    Runs SQL commands against the Hive tables in the Data Lake using Hive's
-    command line interface.
+    [DEPRECATED] Formerly ran SQL commands against the Hive tables in the Data Lake
```

Comment:
We should also log a message about the deprecation if this function is used, similar to https://github.com/wikimedia/wmfdata-python/blob/master/wmfdata/mariadb.py#L148

Also, can you update the `load_csv` function to use `run` instead of `run_cli`, on lines 144 and 145 below?

Reply:
> Can you point me to the discussion on Phabricator? I don't see it in [T275233](https://phabricator.wikimedia.org/T275233). We should probably get that done before merging this to the main branch, and definitely before releasing this in a new version.

I tried to find it, and couldn't, I'm sorry.  Luca pointed me to it, I'll ask him.  But more importantly, yeah, let's get Andrew or someone to just set up the environment.

Reply:
done, and done

Reply:
Thanks! It would also be good if the deprecation message advises using `run` instead.